benchmarks/config/base/base.json

```diff
@@ -60,6 +67,13 @@
       "flags": [ "-n", "100" ],
       "extensions": [ "avx512.*" ]
     },
+    "f32_flat_3x1024_unp_mlir": {
```

Comment:
We really need to be moving off hand-crafted MLIR files and use `mlir-gen` for everything.

Reply:
No, just move it up and keep a consistent naming. I'll refactor this in one single PR later.

Reply:
moved up, the name is consistent with the other unpack we have, no?

Reply:
one nit `fp32`, otherwise ok.